packages/core/src/lib/implementation/persist.unit.test.ts

```diff
@@ -41,7 +41,7 @@ describe('persistReport', () => {
     });
 
     const jsonReport: Report = JSON.parse(
-      readFileSync(join(MEMFS_VOLUME, 'report.json')).toString(),
+      await readFile(join(MEMFS_VOLUME, 'report.json'), 'utf8'),
```

Comment:
Today I found out how to get `readFile` to resolve to a `string` directly instead of a `Buffer` :grin: 

> If no encoding is specified (using `options.encoding`), the data is returned as a `Buffer` object. Otherwise, the data will be a string.